src/validatedODEs.jl

```diff
@@ -119,7 +119,7 @@ end
 #     for its = 1:10
 #         # Remainder of Picard iteration
 #         Δ = picard_remainder!(f!, t, x, dx, xxI, dxxI, δI, δt, Δx, Δ0, params)
-# 
+#
```

Comment:
this and the other empty character changes are automatic from my text editor, sorry for the noise..